app/controllers/navigation/Navigator.scala

```diff
@@ -230,14 +234,15 @@ object Navigator {
   }
 
   val occasionalItemPage: PartialFunction[DeclarationPage, (Mode, String) => Call] = {
-    case PackageInformation            => controllers.declaration.routes.NactCodeSummaryController.displayPage
-    case AdditionalInformationRequired => controllers.declaration.routes.PackageInformationSummaryController.displayPage
-    case AdditionalInformationSummary  => controllers.declaration.routes.PackageInformationSummaryController.displayPage
-    case CusCode                       => controllers.declaration.routes.UNDangerousGoodsCodeController.displayPage
-    case NactCode                      => controllers.declaration.routes.NactCodeSummaryController.displayPage
-    case NactCodeFirst                 => controllers.declaration.routes.TaricCodeSummaryController.displayPage
-    case CommodityMeasure              => controllers.declaration.routes.PackageInformationSummaryController.displayPage
-    case page                          => throw new IllegalArgumentException(s"Navigator back-link route not implemented for $page on occasional")
+    case PackageInformation                => controllers.declaration.routes.NactCodeSummaryController.displayPage
+    case AdditionalInformationRequired     => controllers.declaration.routes.PackageInformationSummaryController.displayPage
+    case AdditionalInformationSummary      => controllers.declaration.routes.PackageInformationSummaryController.displayPage
+    case CusCode                           => controllers.declaration.routes.UNDangerousGoodsCodeController.displayPage
+    case NactCode                          => controllers.declaration.routes.NactCodeSummaryController.displayPage
+    case NactCodeFirst                     => controllers.declaration.routes.TaricCodeSummaryController.displayPage
+    case CommodityMeasure                  => controllers.declaration.routes.PackageInformationSummaryController.displayPage
+    case AdditionalFiscalReferencesSummary => controllers.declaration.routes.ProcedureCodesController.displayPage
```

Comment:
This line seems to be the same for all journeys. Should that not be put into `commonItem` partial function? I'd expect it to be similar to `FiscalInformation`